db/schema.rb

```diff
@@ -54,9 +54,9 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
-  add_foreign_key "comments", "posts", column: "posts_id"
+  add_foreign_key "comments", "posts"
   add_foreign_key "comments", "users", column: "author_id"
-  add_foreign_key "likes", "posts", column: "posts_id"
+  add_foreign_key "likes", "posts"
```

Comment:
- [ ] Kindly fix this issue in the comment table.  the problem is coming in a migration, check this :point_down:  for more info.
![image](https://user-images.githubusercontent.com/65068771/154371157-4b0b476d-bb42-4dac-b7ff-ec811e01b029.png)